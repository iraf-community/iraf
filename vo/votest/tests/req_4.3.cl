#
#  ServiceURLs and resource names are interchangeable.

# Set the test description string.
votest.descr = "ServiceURLs and resource names are interchangeable."

print ("------------------------------------------------------------------")
print ("Req  4.3:  Users shall be able to specify a base ServiceURL to")
print (" 	   data-access tasks expecting a Resource parameter.")
print ("------------------------------------------------------------------")


fcache init

# Execute the test commands.

#  Get the base NED ServiceURL, use that to query a catalog service rather
#  than the name of the service.

regdb ("resolve", "veron")			
getcat (regdb.url, field="3c273", size=0.005, output="STDOUT", format="raw")

