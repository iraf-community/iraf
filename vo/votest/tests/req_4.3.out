------------------------------------------------------------------
Req  4.3:  Users shall be able to specify a base ServiceURL to
 	   data-access tasks expecting a Resource parameter.
------------------------------------------------------------------
<?xml version="1.0" encoding="UTF-8"?><VOTABLE version="1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ivoa.net/xml/VOTable/v1.1  http://www.ivoa.net/xml/VOTable/v1.1" xmlns="http://www.ivoa.net/xml/VOTable/v1.1"><DEFINITIONS><COOSYS equinox="2000" system="eq_FK5"></COOSYS></DEFINITIONS><RESOURCE ID="T9001"><DESCRIPTION>
       HEASARC Browse data service
       Please send inquiries to mailto:request@athena.gsfc.nasa.gov
    </DESCRIPTION><PARAM ucd="OBS_ANG-SIZE" ID="default_search_radius" value="0.0333333333333333" datatype="char" arraysize="*"></PARAM><TABLE><DESCRIPTION> Veron Catalog of Quasars & AGN, 13th Edition </DESCRIPTION><FIELD name="unique_id" datatype="int" arraysize="*"><DESCRIPTION> Integer key </DESCRIPTION></FIELD><FIELD ucd="ID_MAIN" name="name" datatype="char" arraysize="*"><DESCRIPTION> Most Common Designation of the Object </DESCRIPTION></FIELD><FIELD ucd="POS_EQ_RA_MAIN" unit="degree" name="ra" datatype="double" arraysize="*"><DESCRIPTION> Right Ascension </DESCRIPTION></FIELD><FIELD ucd="POS_EQ_DEC_MAIN" unit="degree" name="dec" datatype="double" arraysize="*"><DESCRIPTION> Declination </DESCRIPTION></FIELD><FIELD ucd="phot.mag;em.opt" unit="mag" name="app_mag" datatype="double" arraysize="*"><DESCRIPTION> Magnitude (Type Specified by app_mag_flag) </DESCRIPTION></FIELD><FIELD name="app_mag_flag" datatype="char" arraysize="*"><DESCRIPTION> Magnitude Type: V=visual,P=photographic,B=blue,R=red,I/J/K=infrared,O=O/J-plate </DESCRIPTION></FIELD><FIELD name="redshift_flag" datatype="char" arraysize="*"><DESCRIPTION> Flag for Redshift: E=estimate, >=lower limit </DESCRIPTION></FIELD><FIELD ucd="src.redshift" name="redshift" datatype="double" arraysize="*"><DESCRIPTION> Redshift </DESCRIPTION></FIELD><FIELD ucd="src.class" name="object_type" datatype="char" arraysize="*"><DESCRIPTION> Veron Object Classification </DESCRIPTION></FIELD><FIELD unit="'" name="Search_Offset" datatype="double" arraysize="*"><DESCRIPTION> Offset of target/observation from query center </DESCRIPTION></FIELD><DATA><TABLEDATA><TR><TD>73432</TD><TD>3C 273.0</TD><TD>187.2779</TD><TD>2.0522</TD><TD>12.85</TD><TD>V</TD><TD></TD><TD>0.158</TD><TD>S1.0</TD><TD>0.011</TD></TR></TABLEDATA></DATA></TABLE></RESOURCE></VOTABLE>
